src/github.rs

```diff
@@ -0,0 +1,272 @@
+use crate::install::AssetType;
```

Comment:
What about license header. Have we moved over to skipping that? (That's a fully valid option as it seems.)